runtime/rasdump/javadump.cpp

```diff
@@ -2435,17 +2485,12 @@ JavaCoreDumpWriter::writeThreadsWithNativeStacks(void)
 
 			RasDumpGlobalStorage *dumpGlobals = (RasDumpGlobalStorage *)_VirtualMachine->j9rasdumpGlobalStorage;
 
-			switch (dumpGlobals->showNativeSymbols) {
-			case J9RAS_JAVADUMP_SHOW_NATIVE_STACK_SYMBOLS_ALL:
+			if (J9_ARE_ALL_BITS_SET(dumpGlobals->dumpFlags, J9RAS_JAVADUMP_SHOW_NATIVE_STACK_SYMBOLS_ALL)) {
```

Comment:
When testing a single bit, please use `J9_ARE_ANY_BITS_SET()` instead of `J9_ARE_ALL_BITS_SET()`.

Reply:
What is the reason for using `J9_ARE_ANY_BITS_SET` over `J9_ARE_ALL_BITS_SET` on single bits? I see there is a mixed use of both in existing code.

Reply:
`J9_ARE_ANY_BITS_SET` is (slightly) less expensive than `J9_ARE_ALL_BITS_SET`. When testing exactly one bit, there's no need to use the more expensive macro.